docs/source/hints.mdx

```diff
@@ -2,9 +2,11 @@
 title: Composition hints
 ---
```

Comment:
I opted not to include a minVersion in the frontmatter since Composition hints were available before 2.4 via the `rover supergraph compose` command.